README.md

```diff
@@ -1,70 +1,61 @@
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+### Mobile view
+<img width="400" height="550" alt="Screenshot" src="https://user-images.githubusercontent.com/106375522/216602276-7ff7d5bd-46f6-467f-a505-e1dd890ae1a7.png">
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Quick Start and Commands
 
-### `npm test`
+1. Clone the repo:
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```bash
+git clone https://github.com/yan-fung/react-technical-test.git
+```
 
-### `npm run build`
+2. To install the package locally:
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+```bash
+npm install
+```
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+3. Run the development server for the app.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+```bash
+npm start
+```
 
-### `npm run eject`
+4. To test the codes in tests folder. 
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+```bash
+npm test
+```
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Technology
+- [Jest](https://jestjs.io/docs/getting-started) and [React Testing library](https://testing-library.com/docs/react-testing-library/intro/) are used for testing purposes.
+- [Axios](https://www.npmjs.com/package/axios) is used to perform HTTP requests for retrieving data from the API
+- [PropTypes](https://www.npmjs.com/package/prop-types) is used for validating props.
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+## How To Use
+Enter the name of the stars or planets in the search bar to get the images.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## Improvements & Further Development
+If I would have more time, I would like to add the following features to my application:
+- Making the application more repsonsive to different sizes of device.
+- Allowing users to see some basic info regarding the stars/planets alongside the images.
```

Comment:
Could add another stretch goal here to include some meaningful `alt` tags to the images to improve accessibility for users needing screen readers